### ISSUE TITLE
Accessibility check

### ISSUE THREAD

**Issue by daaain** [ustwo/ustwo.com-frontend#203]
After doing the markup basics in #192 we should have a review of how we're doing in terms of accessibility.

There's a nice tool to get a basic idea how React components are conforming: https://github.com/rackt/react-a11y

**Comment by phil-linnell** [comment]
Requires a little more research and refactoring first

**Comment by phil-linnell** [comment]
Closed as reassigning to new issue tracker.
